src/main/java/tahub/contacts/ui/MainWindow.java

```diff
@@ -178,8 +178,9 @@ private CommandResult executeCommand(String commandText) throws CommandException
             logger.info("Result: " + commandResult.getFeedbackToUser());
             resultDisplay.setFeedbackToUser(commandResult.getFeedbackToUser());
 
-            // Immediately refresh the person list panel after enrollment/unenrollment commands
-            if (commandText.startsWith("enroll") || commandText.startsWith("unenroll")) {
+            // Immediately refresh person list panel after course deletion or enrollment changes
+            if (commandText.startsWith("deletec") || commandText.startsWith("enroll") ||
```

Comment:
What about we do ```refreshPersonView()``` after __every__ command, so there is no need to think so much. Currently you are missing out __edit__ commands and there is actually no ```delete``` command actually...